tests/test_pytest_cov.py

```diff
@@ -1220,11 +1224,17 @@ def target_fn():
 
 def test_run_target():
     p = multiprocessing.Process(target=target_fn)
-    p.start()
-    time.sleep(0.5)
-    event.wait(1)
-    p.terminate()
-    p.join()
+    try:
+        p.start()
+        time.sleep(0.5)
+        event.wait(1)
+    except:
+        p.terminate()
+        raise
+    else:
+        p.close()
```

Comment:
There is no Process.close method?

Reply:
https://docs.python.org/3/library/multiprocessing.html#multiprocessing.Process.close
It's just new in Python 3.7.

Reply:
I could comment this line, and enable it when all tests use python>=3.7 (maybe few years later...).